utils/refresh-aur-packages.sh

```diff
@@ -0,0 +1,14 @@
+# /bin/bash
+
+source manifest
+pushd "aur-pkgs/" > /dev/null
+# remove all packages
+git rm -- */ > /dev/null || git commit -m "Update AUR packages" > /dev/null
+
+for package in ${AUR_PACKAGES}; do
+    git submodule add --force https://aur.archlinux.org/${package}.git > /dev/null
```

Comment:
You don't need to check them out, just declare them, maybe don't use `--force`?

Anyway, can you separete this script to a different commit? Seems a sef contained change.